.github/workflows/version.yaml

```diff
@@ -0,0 +1,18 @@
+name: bump version
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: '0'
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@1.17.2
```

Comment:
I don't like this action because I don't like it's mechanism - needing to put #major, #minor or #patch. This is metadata which I prefer for more formally specify as I specify other commit metadata. I would prefer this to be:
```
VersionBump: <major|minor|patch>
```
at the end of the commit with the other metadata.

I wonder if we can make our own version of this? Oh, i just found the repo (https://github.com/anothrNick/github-tag-action) and it is shell. We can do it "my" way. What do you think?

Reply:
well by default it bumps the patch version with this setup (and doesn't need anything in the commit message) - I'd like to get that merged..
I don't mind to occasionally put a:
VersionBump: #minor
In the commit message. If the #bothers you, I'm happy to switch over as soon as an alternative is ready.

Reply:
well by default it bumps the patch version with this setup (and doesn't need anything in the commit message) - I'd like to get that merged..
I don't mind to occasionally put a:
VersionBump: #minor
In the commit message. If the #bothers you, I'm happy to switch over as soon as an alternative is ready.

Reply:
It's not that the `#` bothers me, it is that I cannot put the strings anywhere in a commit message without it taking effect. I may need to say #minor for some other reason (maybe I put in a URL that has a #minor anchor), and then it will trigger unless I do something to escape it. I find it obnoxious that software takes over completely what I am able to say. I prefer to require something more explicit. Maybe it's just me (it often is).

Reply:
yeah, I see your point. You are right with this imo.
Would like to switch over as soon as something better is available.

Reply:
use it for now, and i'll see if I can get my forked one going soon and then we can switch. we're not likely to end up with a conflict right away.